automation/e2e-deploy-resources.sh

```diff
@@ -40,7 +40,11 @@ kubectl apply -f "https://github.com/kubevirt/containerized-data-importer/releas
 kubectl apply -f "https://github.com/kubevirt/containerized-data-importer/releases/download/${CDI_VERSION}/cdi-cr.yaml"
 
 # wait for tekton pipelines
-oc rollout status -n openshift-operators deployment/openshift-pipelines-operator --timeout 10m
+kubectl rollout status -n openshift-operators deployment/openshift-pipelines-operator --timeout 10m
+# wait until clustertasks tekton CRD is properly deployed
+while kubectl get crd clustertasks.tekton.dev; [ $? -ne 0 ];do
```

Comment:
How about: `until kubectl get crd clustertasks.tekton.dev; do`? Looks cleaner to me.

Reply:
If you want to have a timeout maybe:

```
timeout 10m bash <<- EOF
  until kubectl get crd clustertasks.tekton.dev; do
    sleep 5
  done
EOF
```